src/TraceEvent/Stacks/ChromiumStackSourceWriter.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using static Microsoft.Diagnostics.Tracing.Stacks.StackSourceWriterHelper;
+
+namespace Microsoft.Diagnostics.Tracing.Stacks.Formats
+{
+    public class ChromiumStackSourceWriter
+    {
+        /// <summary>
+        /// exports provided StackSource to a Chromium Trace File format 
+        /// schema: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/
+        /// </summary>
+        public static void WriteStackViewAsJson(StackSource source, string filePath, bool compress)
+        {
+            if (File.Exists(filePath))
+                File.Delete(filePath);
+
+            using (var writeStream = File.CreateText(filePath))
+                Export(source, writeStream, Path.GetFileNameWithoutExtension(filePath));
+
+            if (compress)
```

Comment:
It looks like PerfView doesn't ever compress the file, but for apps that do, would it make sense to not unconditionally write out the file and then open it and read it back in to compress it?  Presumably, Export could take a compressed stream (or FileStream) and just write to it.

Reply:
that was a very good suggestion, #done